File: Corte1/Python/primero1.py
class Person:
    def __init__(self):
        self.name = ""
        self.age = 0

    def say_hello(self):
        print(f"Hi, I'm {self.name} and I'm {self.age} years old.")
        if (self.age >= 18):
            print(f"{self.name} is an adult. ")
        else:
            print(f"{self.name} is not an adult. ")
               
    def Birthday(self):
        self.age += 1
        print(f"¡Happy Birthday {self.name}!")
        print(f"{self.name} Has turned {self.age}")


def main():
    p1 = Person()
    p1.name = "Ana"
    p1.age = 17
    p1.say_hello()

    p2 = Person()
    p2.name = "Daniel"
    p2.age = 11
    p2.say_hello()

    print("---Test---")
    p1.Birthday()
    p1.say_hello()


if __name__ == "__main__":
    main()

File: Corte1/C++/primero1.cpp
#include <iostream>
#include <string>
using namespace std;
class Person {
public:
string name; // attribute
int age; // attribute
void sayHello() {
cout << "Hi, I'm " << name << " and I'm " << age << " years old.\n";
if (age >= 18) {
	cout<< name << " Is an adult\n";
            }
	else {
		cout<< name << " Is not an adult\n";
            }
}

void Birthday() {
	age++;
	cout << "¡Happy Birthday " << name << "!\n";
	cout << name << " Has Turned "<< age << " Years\n";
}

};
int main() {
	Person p1;
	p1.name = "Ana";
	p1.age = 17;
	p1.sayHello();

	Person p2;
	p2.name = "Daniel";
	p2.age = 11;
	p2.sayHello();

cout<<"---Test---\n";
p1.Birthday();
p1.sayHello();

return 0;
}
